my_game.py

```diff
@@ -90,6 +101,92 @@ def update(self):
             self.kill()
 
 
+class UFOShot(arcade.Sprite):
+    """shot fired by the ufo"""
+
+    def __int__(self, **kwargs):
+        super().__init__(**kwargs)
+
+    def update(self):
+        """update position/kill if out of bounds"""
+
+        self.center_x += self.change_x
+        self.center_y += self.change_y
+
+        self.angle = ((self.change_x / self.change_y) * -90)  # set ange based on our direction
+
+        # kill if out of bounds
+        if self.center_x > SCREEN_WIDTH or self.center_x < 0 and self.center_y > SCREEN_HEIGHT or self.center_y < 0:
+            self.kill()
+
+
+class BonusUFO(arcade.Sprite):
+    """occasionally moves across the screen. Grants the player points if shot"""
+
+    def __int__(self, **kwargs):
+
+        kwargs['filename'] = "images/ufoBlue.png"
+
+        kwargs['scale'] = SPRITE_SCALING
+
+        # set random position off-screen
+        kwargs['center_x'] = random.choice([0, SCREEN_WIDTH])
+        kwargs['center_y'] = random.choice([0, SCREEN_HEIGHT])
+
+        # send arguments upstairs
+        super().__init__(**kwargs)
+
+        self.my_shots = arcade.SpriteList()
+
+        # set random direction
+        if self.center_x > 0:
+            self.change_x = -random.randint(1, UFO_SPEED)
```

Comment:
If UFO_SPEED needs to be an int. You should make sure that it is an int. Maybe cast it here, or make an assert statement in the beginning of the code.

Reply:
actuallly it shouldn't necessarily be an int...